### ISSUE TITLE
Consider surfacing pkgdown warnings as errors in CI

### ISSUE THREAD

**Issue by Bisaloo** [epiverse-trace/packagetemplate#157]
To ensure missing alt text do not go unnoticed.

Follow up from https://github.com/epiverse-trace/blueprints/issues/112#issuecomment-2341240402.
